plugins/outputs/http/README.md

```diff
@@ -28,6 +28,9 @@ format by default.
   # token_url = "https://indentityprovider/oauth2/v1/token"
   # scopes = ["urn:opc:idm:__myscopes__"]
 
+  ## Goole API Auth
+  # credentials_file = "/etc/telegraf/example_secret.json"
```

Comment:
The name `credentials_file` is very generic. If this is a Google Cloud specific feature, could we rename the setting `gcp_credentials_file` or something else that indicates it's only for google apis?

Reply:
This is indeed Google-specific, so yes `gcp_credentials_file` works.

Reply:
Or perhaps `google_application_credentials`. Please let me know your preference.

Reply:
`google_application_credentials` sounds good to me. It's a little long but it's descriptive and it's the same as the environment variable the cli tools use, just lowercase to fit in toml, so it should be familiar for gcp users.

https://cloud.google.com/docs/authentication/getting-started#setting_the_environment_variable

Reply:
Done!